staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -140,7 +140,10 @@ func newWithBackoff(subjectAccessReview authorizationclient.SubjectAccessReviewI
 //       }
 //     }
 //
-func (w *WebhookAuthorizer) Authorize(attr authorizer.Attributes) (authorized bool, reason string, err error) {
+// TODO(mikedanese): We should eventually fail closed when we encounter and
+// error. We are failing open now to preserve backwards compatible behavior.
```

Comment:
fail open/closed needs to be an option... if this is a webhook that doesn't explicitly deny, it's ok to fail with NoOpinion

Reply:
I agree, I'll update the comment. I'm going to implement that when I extend webhook authorizer to be able to return denies.